aiogram/api/client/bot.py

```diff
@@ -167,6 +181,93 @@ async def close(self) -> None:
         """
         await self.session.close()
 
+    @staticmethod
```

Comment:
can we move `staticmethod`s outside of class if they're used internally only? we don't inherit from `Bot` if it's name mangling, do we? :thinking: